### DISCUSSION TITLE
Title for remote

### DISCUSSION THREAD

**Discussion by ianustec** [PRProd/HA-Firemote#21] [Ideas]
Hi,
thank you for this project!

Is it possible to add the device name for the remote?

In a sort of the same way of this image https://i.ibb.co/wy4rj1W/Schermata-2022-12-26-alle-13-24-48.png
![Schermata-2022-12-26-alle-13-24-48](https://user-images.githubusercontent.com/25486599/209708358-378f31c3-41d4-4c89-9b20-d7eeef21d8d6.png)

Thanks

**Comment by PRProd** [comment]
That's a great idea!  Let me play around with a few ideas and I'll get back to you.

**Comment by PRProd** [comment]
@ianustec - What do you think about this as an option?  I think it looks great.
![nameOption1](https://user-images.githubusercontent.com/25486599/209707989-c0cef64f-5992-445d-9424-041006788352.png)

**Reply by ianustec** [reply]
It's beautiful!
But I think it could be better on top because in many cases you don't see the entire picture of the remote in your phone.
But.. maybe it could be an update for future releases.
Well done!

**Comment by PRProd** [comment]
@ianustec - Greetings!  Version [1.10.1](https://github.com/PRProd/HA-Firemote/releases/tag/v1.10.1) has just been released which contains support for optional titles on your firemote cards.  This added feature also offers the ability to choose the position of the title, on the top or on the bottom.

I hope this helps!  Enjoy!

-Doug

**Reply by ianustec** [reply]
Hi

I just tried, awsome!!
